docs/product-installation-guide/src/main/asciidoc/run-dc-standalone-proc.adoc

```diff
@@ -0,0 +1,85 @@
+[id='run-dc-standalone-proc']
+= Running standalone {CENTRAL} 
+
+You can use the {CENTRAL} standalone JAR file to run {CENTRAL} without needing to deploy it to an application server, for example {EAP}. 
+[NOTE]
+====
+Red Hat only supports this installation type when it is installed on premise, on Red Hat Enterprise Linux.
+//Any particular version of RHEL?
+====
+
+.Procedure
+
+. Download the {CENTRAL} standalone JAR file from the https://access.redhat.com[Red Hat Customer Portal].
+. In a terminal window, navigate to the directory where you downloaded the installer file.
+. Create the `application-config.yaml` configuration file with the following contents:
++
+[source]
+----
+swarm:
+  management:
+    security-realms:
+      ApplicationRealm:
+        local-authentication:
+          default-user: local
+          allowed-users: local
+          skip-group-loading: true
+        properties-authentication:
+          path: /path/to/application-users.properties
+          plain-text: true
+        properties-authorization:
+          path: /path/to/application-roles.properties
+datasource:
+  management:
+    wildfly:
+      admin: admin
+----
+. Create the `application-users.properties` file. Include an administrative user and if this {CENTRAL} instance will be a controller for {KIE_SERVER}, include a controller user, for example:
```

Comment:
Forgot to mention that the structure "for example:" comes up several times. Technically IBM calls for complete sentences before colons like this (avoiding things like "the following", etc.). As you know, of course. What I often do is just put "Example" on a new line above the code sample. Makes it pretty scannable as well. I don't particularly care per se. You can keep as is or adjust if you like. Just pointing it out.